src/Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -241,7 +242,8 @@ public function storeOutput(): ?LoadTableQueue
         }
 
         // Check whether or not we are creating typed tables
-        $createTypedTables = in_array(self::TYPED_TABLES_FEATURE, $this->projectFeatures, true);
+        $typedTablesFeatures = [self::TYPED_TABLES_FEATURE, self::NATIVE_TYPES_FEATURE];
+        $createTypedTables = count(array_intersect($this->projectFeatures, $typedTablesFeatures)) === 2;
```

Comment:
Radeji bych dal 2x `in_array`, tohle je trosku cryptic :smile:

Reply:
Zrovna jsem chtěl jedovatě napsat něco ve smyslu, jestli by to nešlo třeba base64 encodnout a přes checknout to přes eval(base64decode($featureCheck)) :)